test/string.wisp

```diff
@@ -41,3 +41,22 @@
 
 (is (= ["Some" "words" "to" "split"]
        (split "Some words to split" " ")))
+
+; replace tests
+; basic test
+(is (= "wtring" (replace "string" "s" "w")))
+; testing 'g' flag for replace
+(is (= "hewwo" (replace "string" "l" "w")))
```

Comment:
I think there is a typo here, I' imagine you meant `(replace "hello" "l" "w")` instead.